src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -288,7 +289,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
                 {showSoundCloudRadio && (
                   <S.RadioTrigger>
-                    <div
+                    <button
```

Comment:
I keep seeing `buttons` without any onClick prop what's the benefit here?

Reply:
@denisdimitrov buttons should have the underlying onClicks. The benefit is so that the markup is aligned with the functionality. If it can be clicked, it should be a button

Reply:
https://github.com/sparkletown/sparkle/pull/1727/files/b708780c5afa1dc9f135af4bc00a12d914d47e59#diff-d2dbbd9743869736d76cf73512c615856022b7a3ee80ec9b63bed997c40f7e37R296